pkg/virt-handler/isolation/isolation.go

```diff
@@ -296,7 +296,7 @@ func (r *realIsolationResult) MountInfoRoot() (*MountInfo, error) {
 			}
 		}
 
-		if record[3] == "/" && record[4] == "/" {
+		if record[4] == "/" {
 			return &MountInfo{
```

Comment:
Besides checking in MountInfoRoot(), we also need fix following codes in Mount() to add nodeMountInfo.Root into sourceFile as well.
```
				sourceFile, err := containerdisk.GetImage(filepath.Join(nodeRes.MountRoot(), nodeMountInfo.MountPoint), volume.ContainerDisk.Path)
				if err != nil {
					return fmt.Errorf("failed to find a sourceFile in containerDisk %v: %v", volume.Name, err)
				}
```